### DISCUSSION TITLE
What diffrent betweek Cosmos Userkit and Cosmos Devkit?

### DISCUSSION THREAD

**Discussion by mrunknown850** [CosmosOS/Cosmos#1884] [Q&A]
I normally think the name Userkit is for non developer user what to make an OS without advance knowlege about C# and Devkit is for advance developer want to make advance OS like an Windows 7 replica or sth. But now when I use both version, I think they are the same. Is there anything diffrent?

**Comment by zarlo** [comment]
Userkit is most of the time older and more stable and prebuilt
devkit is build from source less likely to be stable and is what is used to work on cosmos it self
